controllers/managedocs_controller.go

```diff
@@ -509,8 +517,7 @@ func (r *ManagedOCSReconciler) reconcilePhases() (reconcile.Result, error) {
 		if err := r.reconcileOCSInitialization(); err != nil {
 			return ctrl.Result{}, err
 		}
-
-		// fix for non-converged deployment will be provided in upcoming PR
+		// Fix for non-converged deployment type will be provided in upcoming PR
 		if r.DeploymentType == "converged" {
```

Comment:
Please use constants for deployment types ("converged", " consumer", "provider") - this was already asked in a previous review